python/py-pytest-forked/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pytest-forked
+version             1.0.1
+categories-append   devel
+platforms           darwin
+license             MIT
+
+python.versions     27 34 35 36 37
```

Comment:
Are py34, py35 and py36 needed as a dependency for an existing port(s)? Otherwise, please remove these old sub-ports.

Reply:
Never mind. I just read your commit message. They are.